PopApp/Navigation/Navigation.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import {createMaterialTopTabNavigator } from '@react-navigation/material-top-tabs'
+import Launch from '../Components/Launch'
+import Home from '../Components/Home'
+import Connect from '../Components/Connect'
+import {Dimensions} from 'react-native'
+import { NavigationContainer } from '@react-navigation/native';
+import { SafeAreaView } from 'react-native-safe-area-context'
+
+
+const TopTabNavigator = createMaterialTopTabNavigator()
+
+export default function App() {
```

Comment:
Does this function control the main execution of the app? If so, please comment.